src/main/java/de/tum/in/www1/artemis/service/UserService.java

```diff
@@ -146,8 +149,15 @@ public StandardPBEStringEncryptor encryptor() {
         return encryptor;
     }
 
-    @Value("${artemis.encryption-password}")
-    private String ENCRYPTION_PASSWORD;
+    /**
+     * Returns the password of the user as a char (as is the Java convention for handling sensitive Strings)
```

Comment:
A bit of a nit-pick here, but it should be char array

Reply:
Changed